apps/designer/app/designer/features/style-panel/controls/font-weight/font-weight-control.tsx

```diff
@@ -2,7 +2,7 @@ import { Select } from "@webstudio-is/design-system";
 import { FontWeight, fontWeights } from "@webstudio-is/fonts";
 import { toValue } from "@webstudio-is/css-engine";
 import { useMemo } from "react";
-import { useAssets } from "~/designer/shared/assets";
+import { useAssetContainers } from "~/designer/shared/assets";
```

Comment:
I guess we could have still called it `useAsset` as a name, because asset is still in there, it doesn't dictate that you can receive asset container object